beacon-chain/state/v3/deprecated_setters.go

```diff
@@ -0,0 +1,31 @@
+package v3
+
+import (
+	"github.com/pkg/errors"
+	ethpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+)
+
+// SetPreviousEpochAttestations is not supported for HF1 beacon state.
+func (b *BeaconState) SetPreviousEpochAttestations(val []*ethpb.PendingAttestation) error {
+	return errors.New("SetPreviousEpochAttestations is not supported for hard fork 1 beacon state")
```

Comment:
This would be hard fork 2 , no ? Maybe better to just mention it by the fork name.